### PR TITLE
adding gffcompare

### LINKED ISSUE #3041: new module: gffcompare
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by jemten**
## PR checklist

Closes #3041  

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/gffcompare/main.nf [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+    conda "bioconda::gffcompare=0.12.6"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gffcompare:0.12.6--h9f5acd7_0' :
+        'quay.io/biocontainers/gffcompare:0.12.6--h9f5acd7_0' }"
+
+    input:
+    tuple val(meta), path(gtfs)
+    tuple val(meta2), path(fasta), path(fai)
+    path reference_gtf
```
Comment (matthdsm) [inline modules/nf-core/gffcompare/main.nf:13]: If the reference is part of a reference dataset, could you also add meta here?

**Review by matthdsm — APPROVED** [review]
LGTM, apart from the `meta` nitpicking

File: modules/nf-core/gffcompare/main.nf (hunk 1 above)
Comment (jemten) [reply modules/nf-core/gffcompare/main.nf:13]: Good point! I'll add meta to this input as well

Comment (jemten) [reply modules/nf-core/gffcompare/main.nf]: Good point! I'll add meta to this input as well

**Pull request merged**